### PR TITLE
Adding nginx config as a separated configmap to make it more configurable

### PR DESCRIPTION
**Pull request by rajdas98**
Signed-off-by: rajdas98 <mail.rajdas@gmail.com>

## Proposed changes

Currently, the server addresses are hardcoded in the frontend docker image, so that it is hard to configure the endpoints. Adding nginx config as a separate configmap and mount it with the frontend deployment.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by ishangupta-ds — APPROVED** [review]

File: litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml [hunk 1]
```diff
@@ -476,6 +476,68 @@ data:
...
+
+        location /api/ {
+            proxy_set_header   Host                 $host;
+            proxy_set_header   X-Real-IP            $remote_addr;
+            proxy_set_header   X-Forwarded-For      $proxy_add_x_forwarded_for;
+            proxy_set_header   X-Forwarded-Proto    $scheme;
+            proxy_pass "http://litmusportal-server-service:9002/";
+        }
+
+        location /ws/{
```
Comment (Jasstkn) [inline litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml:530]: add space here for consistent syntax.

File: litmus-portal/namespaced-k8s-template.yml [hunk 2]
```diff
@@ -476,6 +476,68 @@ data:
...
+
+        location /api/ {
+            proxy_set_header   Host                 $host;
+            proxy_set_header   X-Real-IP            $remote_addr;
+            proxy_set_header   X-Forwarded-For      $proxy_add_x_forwarded_for;
+            proxy_set_header   X-Forwarded-Proto    $scheme;
+            proxy_pass "http://litmusportal-server-service:9002/";
+        }
+
+        location /ws/{
```
Comment (Jasstkn) [inline litmus-portal/namespaced-k8s-template.yml:530]: here too

File: litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/platforms/okteto/litmus-portal-dev-manifest-template.yml]: done

File: litmus-portal/namespaced-k8s-template.yml (hunk 2 above)
Comment (rajdas98) [reply litmus-portal/namespaced-k8s-template.yml]: done

**Review by gdsoumya — APPROVED** [review]

**Pull request merged**